x-pack/plugins/lens/public/assets/apply_changes_illustration.tsx

```diff
@@ -0,0 +1,553 @@
+/*
```

Comment:
this file made the whole async bundle increase over 100KB... Could we simplify it? I'll try to paste it to svgomg and see 👁️

Reply:
Thank you for working on this, @mbondyra ! I appreciate it.

Reply:
@mbondyra, @andrewctate: The illustration SVG file size appeared close in size to the other SVG illustrations we already use in Lens (such as the drop fields illustration). But I'd be happy to cut two PNGs (at 2x resolution) for light and dark mode, if that would be preferable and reduce the bundle size. Stay tuned.